src/hvsampledata/__init__.py

```diff
@@ -621,6 +631,11 @@ def nyc_taxi_remote(
     Load specific columns:
 
     >>> df = nyc_taxi_remote("polars", engine_kwargs={"columns": ["pickup_x", "pickup_y"]})
+
+    Download without loading into memory:
+
+    >>> path = nyc_taxi_remote("pandas", download_only=True)
```

Comment:
```suggestion
    >>> path = nyc_taxi_remote("download")
```

I still like something like this, I don't think it needs to specify backend if it isn't used at all.

Reply:
It makes sense, but I'm also thinking that the `engine` parameter now no longer means specifying a dataframe engine but also a specific action (like download only). Isn't that a bit confusing?